spec/components/search_spec.rb

```diff
@@ -576,7 +621,10 @@ def search
     end
 
     it 'can use category as a search context' do
-      category = Fabricate(:category)
+      category = Fabricate(:category,
+        search_priority: Searchable::PRIORITIES[:ignore]
+      )
+
```

Comment:
it is fine to sit on this ... but also test this returns stuff. 

```
 search = Search.execute("hello #{category.slug}")
```

Reply:
O this is already tested in https://github.com/discourse/discourse/blob/0a8f95028176c62b06194804be23bfd944ac5873/spec/components/search_spec.rb#L874